appconfiguration/spring-cloud-azure-feature-management-web/spring-cloud-azure-feature-management-web-sample/src/main/resources/application.yml

```diff
@@ -9,7 +9,7 @@ spring:
           delay: 5s
```

Comment:
Can you add a readme explaining how to run and what is happening?

Reply:
Sure. Added README.md